Tools/parametric_model/generate_parametric_model.py

```diff
@@ -127,5 +127,7 @@ def start_model_estimation(config, log_path, data_selection="none", plot=False):
                         help='Configuration file path for pipeline configurations')
     parser.add_argument('--plot', metavar='plot', type=str2bool, default='True',
                         help='Show plots after fit.')
+    parser.add_argument('--normalization', metavar='normalization', type=str2bool, default='True', required=False,
```

Comment:
I think the default should be false, since this would break all the other models

Reply:
If I understood it correctly, normalization was always applied whenever the topic "V_air_body_x" was missing from the data frame (line 87ff in /Tools/parametric_model/src/models/dynamics_model.py). Maybe we could remove this however, if we don't want to run simulation files through this pipeline or add the missing topics already to the csv-file before.

Reply:
Normalization can be removed as soon as all models (and possibly the logs as well) have been adapted to include vehicle_torque_setpoints and thrust_setpoints (already normalized). Normalization is only required to obtain these values from the actuator outputs.